### PR TITLE
fix blacklist not ignoring owner

### PR DESCRIPTION
**Pull request by QuantumlyTangled**
### Description of the PR

This PR just fixes the blacklist inhibitor not ignoring the bot owner.

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

-

### Semver Classification

- [ ] This PR only includes documentation or non-code changes.
- [x] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [ ] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

File: src/inhibitors/blacklist.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const { Inhibitor } = require('klasa');
 module.exports = class extends Inhibitor {
 
 	async run(msg) {
+		if (msg.author === this.client.owner) return;
```
Comment (kyranet) [inline src/inhibitors/blacklist.js]: For performance, I'd suggest `this.client.options.ownerID`, as `this.client.owner` is a getter:

https://github.com/dirigeants/klasa/blob/bb46f404a40448c880f0c0d4662a894f27dc7977/src/lib/Client.js#L296-L298

**Comment by Soumil07** [comment]
What if the owner hasn't passed ownerID in his clientOptions?

**Comment by MrJacz** [comment]
https://github.com/dirigeants/klasa/blob/master/src/lib/Client.js#L408

**Comment by Soumil07** [comment]
Oh. I thought it did that in the getter.

**Comment by QuantumlyTangled** [comment]
In that case, you might also want to change other inhibitors.

**Comment by QuantumlyTangled** [comment]
Or I can.

**Comment by bdistin** [comment]
In node 9 a get is as fast as a property access. And a reference compare has always been faster than a string compare. The way this pr was originally done (msg.author === this.client.owner) is the correct way to perform the check.

**Pull request merged**